File: iOSbeautyOfDevelop/chapter11/11.9/TableViewInViewController/TableViewInViewController/ViewController.m
//
//  ViewController.m
//  TableViewInViewController
//
//  Created by aa64mac on 9/3/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    self.radioListArray = @[@"CRI怀旧金曲", @"CRI轻松调频", @"CRI新闻广播", @"CRI劲歌FM88.7"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
    return [self.radioListArray count];
}

- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"radio cell" forIndexPath:indexPath];
    cell.textLabel.text = self.radioListArray[indexPath.row];
    return cell;
}

@end

File: iOSbeautyOfDevelop/chapter11/11.9/TableViewInViewControllerS/TableViewInViewControllerS/ViewController.swift
//
//  ViewController.swift
//  TableViewInViewControllerS
//
//  Created by aa64mac on 9/3/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {

    var radioListArray = ["CRI怀旧金曲", "CRI轻松调频", "CRI新闻广播", "CRI劲歌FM88.7"]
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return radioListArray.count;
    }
    

    
    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
        var cell: UITableViewCell = tableView.dequeueReusableCellWithIdentifier("cell1", forIndexPath: indexPath) as! UITableViewCell
        cell.textLabel?.text = radioListArray[indexPath.row]
        return cell
    }

}

